superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/FiltersConfigForm.tsx

```diff
@@ -352,8 +344,8 @@ const FiltersConfigForm = (
   // @ts-ignore
   const hasDataset = !!nativeFilterItems[formFilter?.filterType]?.value
     ?.datasourceCount;
-  const hasColumn =
-    hasDataset && !FILTERS_WITHOUT_COLUMN.includes(formFilter?.filterType);
+  const hasColumn = hasDataset;
```

Comment:
I think we don't need anymore `hasColumn`, you can remove it and use in every place `hasDataset`

Reply:
Agreed, let's remove it if it's not needed anymore